functions/src/controllers/AdminController.ts

```diff
@@ -208,26 +225,339 @@ const Admin = {
+                        orderObj.user_id = userObj.id
+                        
+                        orderObj.order_no = Order.getOrderNos(orderObj.payment_id, 0) //return order no
+                        orderObj = PaymentGateway.getOrdersObj(orderObj);
+                        // create Payment object
+                        paymentObj.user_id = userObj.id
+                    } catch (error) {
+                        console.log(error);
+                        return res.status(500).send({ error })
+                    }
+                    Admin.saveOrdersUnderUser(orderObj, paymentObj, userStore[orderObj.shipping_address.phone])
+                    ordermaster.push(orderObj)
+                }
+
+                return res.status(200).send({ message: "Upload is in progress please check airtable." })
+            });
+        } catch (e) {
+            res.status(500).send({ message: "Something went wrong" })
+        }
+    },
+
+    getUser: (payload) => {
+        console.log("checkInDb===>");
+        return new Promise(async (resolve, reject) => {
+
+            const firestore = admin.firestore();
+            const userDocs = await firestore.collection('user-details').where('phone', "==", payload.phone.replace("+91","")).get();
+            const size = userDocs.size
+            
+            if (size) {
+                let userObj:any = false
+                for (let i = 0; i < size; i++) {
+                    const userData = userDocs.docs[i].data()
+                    
+                    if (userData.verified) {
+                        userObj ={id: userDocs.docs[i].id, ...userData} 
+                        break;
+                    }
+                }
+                if (userObj) {
+                    resolve(userObj)
+                } else {
+                    resolve({id: userDocs.docs[0].id, ...userDocs[0].data()} )
+                }
+            } else {
+                try {
+                    
+                    let newUser  = await Admin.createVerifiedUser(payload)
+                    resolve(newUser)
+                } catch (error) {
+                    reject(false)
+                }
+            }
+        })
+    },
+    createVerifiedUser: function (payload) {
+        console.log("create user", payload.phone);
         
+        return new Promise(async (resolve, reject) => {
+            try {
+                const newUser = await admin.auth().createUser({
+                    phoneNumber: payload.phone
+                })
+
+                await admin.firestore().collection("user-details").doc(newUser.uid).set({
+                    verified: true,
+                    ...payload,
+                    phone: payload.phone.replace("+91", "")
+                })
+                resolve({
+                    id: newUser.uid,
+                    verified: true,
+                    ...payload,
+                    phone: payload.phone.replace("+91", "")
```

Comment:
why is +91 removed here?

also, this is used too frequently can go in formatPhoneNumber() method